app/src/main/java/com/github/partymakers/partymaker/party/CreatePartyActivity.java

```diff
@@ -0,0 +1,252 @@
+package com.github.partymakers.partymaker.party;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.app.DatePickerDialog;
+import android.app.TimePickerDialog;
+import android.content.Context;
+import android.os.Bundle;
+import android.util.TypedValue;
+import android.view.KeyEvent;
+import android.view.View;
+import android.view.inputmethod.EditorInfo;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.CompoundButton;
+import android.widget.DatePicker;
+import android.widget.Switch;
+import android.widget.TimePicker;
+
+import com.github.partymakers.partymaker.R;
+import com.github.partymakers.partymaker.databinding.ActivityCreatePartyBinding;
+import com.google.android.material.chip.Chip;
+import com.google.android.material.chip.ChipDrawable;
+import com.google.android.material.chip.ChipGroup;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.List;
+
+public class CreatePartyActivity extends AppCompatActivity implements View.OnClickListener {
+    private int year, month, day, hour, minute;
+    private ActivityCreatePartyBinding viewBinding;
+    Switch dressCode = null;
+    private List<String> tagListFood = Arrays.asList("Polish", "Pizza", "Kebab", "Sushi", "Asian", "Italian", "Burgers", "Mexican", "Vietnamese"); //make it final?
+    private List<String> tagListDrinks = Arrays.asList("Coke", "Sprite", "Fanta", "Beer", "Warka", "Heineken", "Vodka", "Whiskey", "Martini", "Shots");
```

Comment:
Don't include trademarks. Use categories like "light beer", "craft beer", "cocktails", "spirits", "soft drinks", "juices", "tea", "coffee" etc. Be neither too specific nor too broad.

Reply:
Done ✅